samples/zigbee/light_bulb/src/main.c

```diff
@@ -505,6 +517,7 @@ void main(void)
 
 	/* Initialize */
 	configure_gpio();
+	settings_subsys_init();
```

Comment:
Should we check returned value?

Reply:
added LOG_ERR if returned value is non-zero